src/store/soraCard/types.ts

```diff
@@ -9,4 +9,5 @@ export type SoraCardState = {
   totalXorBalance: FPNumber;
   xorToDeposit: FPNumber;
   totalXorBalanceUpdates: Nullable<Subscription>;
+  authLogin: any;
```

Comment:
please define it

Reply:
everywhere

Reply:
It's Paywings module. It's written in pure JS, they don't provide ts typings